scripts/python/examples/read_met_point_obs.py

```diff
@@ -0,0 +1,61 @@
+'''
+Created on Nov 10, 2021
+
+@author: hsoh
+
+This script reads the MET point observation NetCDF file like MET tools do.
+
+Usage:
+
+  python3 read_met_point_obs.py <MET_point_observation_NetCDF_name>
+  python3 read_met_point_obs.py <space_separated_text_file>
+      <space_separated_text_file>: 11 columns
+          'typ', 'sid', 'vld', 'lat', 'lon', 'elv', 'var', 'lvl', 'hgt', 'qc', 'obs'
+           string columns: 'typ', 'sid', 'vld', 'var', , 'qc'
+          numeric columns: 'lat', 'lon', 'elv', 'lvl', 'hgt', 'qc', 'obs'
+
+'''
+
+import os
+import sys
+from datetime import datetime
+
+from met.point import met_point_tools
+from pyembed.python_embedding import pyembed_tools
+
+ARG_PRINT_DATA = 'show_data'
+DO_PRINT_DATA = ARG_PRINT_DATA == sys.argv[-1]
+
+start_time = datetime.now()
+
+point_obs_data = None
+input_name = sys.argv[1]
+prompt = met_point_tools.get_prompt()
+if len(sys.argv) == 1 or ARG_PRINT_DATA == input_name:
+   # This is an example of creating a sample data
+   point_obs_data = met_point_tools.get_sample_met_point_obs()
```

Comment:
I think this should be `get_sample_point_obs()`, I cannot find `get_sample_met_point_obs()` anywhere in `point.py`.

Reply:
Fixed in 55dc1a96af101a79bf550757935cb111d997b233